.github/workflows/asan_ubsan_build.yml

```
@@ -0,0 +1,47 @@
+
+name: Build with ASAN and UBSAN
+
+on:
+  workflow_dispatch:
+  pull_request: 
+    branches: [trunk]
+    types: [opened, ready_for_review]
+
+env:
+  LDMX_DOCKER_TAG: ldmx/dev:latest
+
+jobs:
+  build-with-asan-ubsan:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout ldmx-sw
+      uses: actions/checkout@v4
+      with:
+        submodules: 'recursive'
+        fetch-depth: 0
+
+    - name: Install just
+      uses: extractions/setup-just@v2
+      with:
+          just-version: 1.26.0
+ 
+    - name: Compile and Install ldmx-sw
+      run: |
+        just install-denv init configure-asan-ubsan 
+        just build
+      shell: bash
+
+    - name: Run ecal_pn config
```

```suggestion
    - name: Run ecal_pn config
      continue-on-error: true
```

We want the job to continue even if this step fails (which it will since ASAN will exit with non-zero if any issues are encountered). https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

---

Perfect thank you! I really was looking for this!

---

You can tell asan to continue on error if needed